File: deaf_grandma.py
def deaf_grandma():

    user_input = input("HEY KID!\n")
    goodbye = True
    count = 0
    while goodbye:
        if user_input == "":
            user_input = input("WHAT?!\n")

        if user_input == "GOODBYE!":
            count+=1
            user_input = input("LEAVING SO SOON?\n")

        if user_input == user_input.lower():
            user_input = input("SPEAK UP, KID!\n")
        elif user_input == user_input.upper() and user_input != "GOODBYE!":
            user_input = input("NO, NOT SINCE 1946!\n")

        if count==1:
            print("LATER, SKATER!")
            goodbye = False
            return
        



deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
// const prompt = require("prompt-sync")();
// var input = prompt("HEY KID!\n");
// console.log(input)


let count = 1;
let input = window.prompt('HEY KID!');
while(count < 2) {

  // if(count ===2) {
  //   alert("LATER, SKATER!");
  // }
  if(input ==="GOODBYE!") {
    input = window.prompt("LEAVING SO SOON?");
    count++;
    alert("LATER, SKATER!");
    break;
  }
  if(input === "") {
    input = window.prompt("WHAT?!");
  }else if(input !== input.toUpperCase()) {
    input = window.prompt("SPEAK UP, KID!");
  }else if(input === input.toUpperCase() && input !== "GOODBYE!" ) {
    input = window.prompt("NO, NOT SINCE 1946!");
  }
}
    
}

deafGrandma();